roles/sap_vm_provision/tasks/platform_ansible/redhat_ocp_virt/execute_provision.yml

```diff
@@ -0,0 +1,347 @@
+    __sap_vm_provision_register_vm_deploy_config:
+      volumes: "{{ storage_disk_name_list + cloud_init_volume + downwardmetrics_volume }}"
+      networks: "{{ __sap_vm_provision_register_networks_definition }}"
+      domain:
+        # shared | auto, auto prevents live migration
+        ioThreadsPolicy: shared
+        hostname: "{{ __sap_vm_provision_register_vm_name }}"
+        evictionStrategy: LiveMigrate
+        terminationGracePeriodSeconds: 1800 # 30 minutes after stop request before VM is force terminated
+        resources:
+          requests:
+            memory: "{{ __sap_vm_provision_register_vm_config.kubevirt_vm_memory_gib + sap_vm_provision_ocp_container_memory_overhead }}Gi" # memory + overhead for container runtime)
+
+        devices:
+          networkInterfaceMultiqueue: true
+          blockMultiQueue: true
+          autoattachMemBalloon: false
+          disks: "{{ storage_disk_list }}"
+          interfaces: "{{ __sap_vm_provision_register_network_interfaces }}"
+
+        cpu:
+          cores: "{{ __sap_vm_provision_register_vm_config.kubevirt_vm_cpu_cores }}"
+          threads: "{{ __sap_vm_provision_register_vm_config.kubevirt_vm_cpu_smt }}"
+          dedicatedCpuPlacement: true
+          model: host-passthrough
+          numa:
+            guestMappingPassthrough: {}
+          features:
+            - name: x2apic
+              policy: require
+            - name: rdtscp
+              policy: require
+            - name: invtsc
+              policy: require
+
+        memory:
+          guest: "{{ __sap_vm_provision_register_vm_config.kubevirt_vm_memory_gib }}Gi"
+          hugepages:
+            pageSize: 1Gi
+
+- name: Provision KubeVirt Virtual Machine
+  kubevirt.core.kubevirt_vm:
+    api_version: "{{ api_version | default(omit) }}"
+    persist_config: "{{ persist_config | default(omit) }}"
+
+    ## Virtual Machine target Hypervisor definition
+    namespace: "{{ sap_vm_provision_ocp_target_namespace }}" # Target namespace
+
+    ## Virtual Machine definition
+    state: present
+    running: true
+    wait: true  # ensure Virtual Machine in ready state before exiting Ansible Task
+    wait_sleep: 30  # 30 second poll for ready state
+    wait_timeout: 600  # 10 minute wait for ready state
+    force: "{{ sap_vm_provision_overwrite_vm | default(false) }}"  # Do not replace existing Virtual Machine with same name
+    name: "{{ __sap_vm_provision_register_vm_name }}"
+    labels:
+      app: "{{ __sap_vm_provision_register_vm_name }}"
+    # Virtual Disk volume definitions
+    data_volume_templates: "{{ storage_disks_map + os_image }}"
+    spec: "{{ __sap_vm_provision_register_vm_deploy_config }}"
+
+- name: Check VM status
+  register: register_provisioned_host_single_info
+  kubevirt.core.kubevirt_vm_info:
+    name: "{{ __sap_vm_provision_register_vm_name }}"
+    namespace: "{{ sap_vm_provision_ocp_target_namespace }}"
+
+- name: Get VMI details
+  kubernetes.core.k8s_info:
```

Comment:
Do we need a `kubevirt.core.kubevirt_vmi_info` module? Or should the VMI status information be added to `kubevirt.core.kubevirt_vm_info`?

Reply:
Where do you see the benefit over using `kubernetes.core.k8s_info`?

Reply:
Just thinking about potential feature requests for `kubevirt.core`. So you don't see any benefit?

Reply:
Intuitively I was expecting that  `kubevirt.core.kubevirt_vm_info` would provide such info. So yes, I think it makes sense to add it there, or provide the VMI info through `kubevirt.core.kubevirt_vmi_info`.